homeassistant/components/airzone/__init__.py

```diff
@@ -0,0 +1,66 @@
+"""The Airzone integration."""
+from __future__ import annotations
+
+from aioairzone.common import ConnectionOptions
+from aioairzone.localapi_device import AirzoneLocalApi
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MANUFACTURER
+from .coordinator import AirzoneUpdateCoordinator
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+class AirzoneDevice(CoordinatorEntity):
+    """Define an Airzone device."""
+
+    def __init__(
+        self,
+        coordinator: AirzoneUpdateCoordinator,
+        zone_id: str,
+        zone_name: str,
+    ) -> None:
+        """Initialize."""
+        super().__init__(coordinator)
+
+        self._attr_device_info: DeviceInfo = {
+            "identifiers": {(DOMAIN, zone_id)},
+            "manufacturer": MANUFACTURER,
+            "name": f"Airzone [{zone_id}] {zone_name}",
+        }
+        self.zone_id = zone_id
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Airzone from a config entry."""
+    options = ConnectionOptions(
+        entry.data[CONF_HOST],
+        entry.data[CONF_PORT],
+    )
+
+    airzone = AirzoneLocalApi(aiohttp_client.async_get_clientsession(hass), options)
+
+    await airzone.update_airzone()
+
```

Comment:
```suggestion
```

`async_config_entry_first_refresh` is already doing this (and will auto retry later if it fails)

Reply:
Done in 9a96604fc814869ecb594340ace7a1325230b05b